magenta/models/score2perf/__init__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Import of Score2Perf problem module."""
+
+from magenta.models.score2perf import score2perf
```

Comment:
are you intentionally wanting to hide the rest of the modules?

Reply:
talked offline, this line is to make sure "from magenta.models import score2perf" registers the t2t problems